Android.mk

```diff
@@ -19,9 +19,48 @@ LOCAL_CPPFLAGS += \
         -Wno-unused-parameter \
         -Wno-int-to-pointer-cast \
         -Wno-missing-field-initializers \
-        -Wno-conversion-null \
-        -Werror \
-        -Wnull-dereference
+	-Wno-conversion-null \
+	-Werror \
+	-Wnull-dereference \
+	-Waddress \
+	-Warray-bounds \
+	-Wc++11-compat \
+	-Wc++14-compat \
```

Comment:
many of the introduced compiler flags to detect possible errors are enabled by default with -Wall usage. Since -Wll usage is already used, check and remove the duplicates

https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html